LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
why not CC BY 3.0 AU or 4.0  as all www.qld.gov.au have https://creativecommons.org/licenses/by/4.0/ links at footer.